README.md

```diff
@@ -98,10 +97,10 @@ and Drop component to a lesson, then click the `EDIT` button.
 ![Edit view](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/edit-view.png)
```

Comment:
@haikuginger my feelings about this mistake: :facepalm:

Of course it won't - I believe it is a version hash. I wonder though, why it was fixed to solutions fork and particular commit. @mtyaka any [hints why|https://github.com/open-craft/xblock-drag-and-drop-v2/commit/7df7a6c77a1d8abf8587db0f63578820e46104da]? Can't we just use relative links, like I've done in the [next commit|https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/75/commits/445fba9c2198a742590d7e77ba44547922266091]?